### ISSUE TITLE
SimpleToast is blocking tvOS preview

### ISSUE THREAD

**Issue by Kyle-Ye** [sanzaru/SimpleToast#32]
For a multi platform projects, we can conditional link it to exclude tvOS in building and also use #canImport in Swift code.

<img width="438" alt="image" src="https://github.com/sanzaru/SimpleToast/assets/43724855/fe2ab430-5b05-4d72-a28d-d5f458c76399">

This works fine for building and archiving. However this will cause an issue when previewing SwiftUI view on tvOS.

```
== PREVIEW UPDATE ERROR:

    SchemeBuildError: Failed to build the scheme ”Forumate”
    
    'DragGesture' is unavailable in tvOS
    
    Compile SimpleToast.swift (arm64):
    /Users/kyle/Library/Developer/Xcode/DerivedData/Forumate-cfcvrozqvmyfrfcrzfasktaukvys/SourcePackages/checkouts/SimpleToast/Sources/SimpleToast/SimpleToast.swift:30:9: error: 'DragGesture' is unavailable in tvOS
            DragGesture()
            ^~~~~~~~~~~
```

**Comment by Kyle-Ye** [comment]
At first, I was thinking adding a PR to add something like `@unavaiable(tvOS)` mark to `SimpleToast`. But later I find we can add tvOS support directly.

There are only 2 issues here:
1. onTapGesture(count:) is supported on tvOS via tvOS 16.0 which is support on Package manifest version 5.8.
2. DragGesture is not available on tvOS. We can just use `#if os(tvOS)` to remove this feature.

If you are fine about it, I'm glad to add a pull request here. @sanzaru

**Comment by sanzaru** [comment]
@Kyle-Ye Thanks for your contribution. I reviewed your PR and left a comment.

**Comment by Kyle-Ye** [comment]
> * onTapGesture(count:) is supported on tvOS via tvOS 16.0 which is supported on Package manifest version 5.7.

onTapGesture(count:) is available on tvOS 16.0 which is supported on Package manifest version 5.7.
